_episodes/03-configuration.md

```diff
@@ -184,71 +209,131 @@ rootpath:
   default: ~/default_inputpath
   CORDEX: ~/default_inputpath
 ```
-Site-specific entries for Jasmin, DKRZ and ETHZ are listed at the end of the
-example configuration file.
 
-In this lesson, we will work with data from
-[CMIP5](https://esgf-node.llnl.gov/projects/cmip5/)
-and [obs4mips](https://esgf-node.llnl.gov/projects/obs4mips/).
-We add the root path of the folder where  our/your data is available.
-
-```yaml
-  rootpath:
-  ...
-    CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2, ~/esmvaltool_tutorial/data]
-    obs4mips: ~/esmvaltool_tutorial/data
-```
+Site-specific entries for Jasmin and DKRZ are listed at the end of the
+example configuration file.
 
-> ## Setting the correct rootpath
+> ## Set the correct rootpath
 >
-> - To get the data (or its correct rootpath), check instruction in
->   [Setup]({{ page.root }}{% link setup.md %}).
-> - For more information about setting the rootpath, see also the ESMValTool
->   [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/esmvalcore/datafinder.html).
-{: .callout}
+> In this tutorial, we will work with data from
+> [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/)
+> and [obs4mips](https://esgf-node.llnl.gov/projects/obs4mips/).
+> How does ESMValTool find the data?
```

Comment:
Very nice that this is now an exercise! I would rephrase this to something more 'active':
```suggestion
> Modify the rootpath section of the tutorial to make sure the data path for the tutorial is set correctly for both CMIP5 and obs4mips. 
```

Reply:
It is revised.